tests/cpp/test_host_irs.cpp

```diff
@@ -377,6 +377,95 @@ TEST_P(HostIrTest, ThreeFusions) {
+  constexpr int64_t kForLoopStart = 1;
+  constexpr int64_t kForLoopStop = 7;
+  constexpr int64_t kForLoopStep = 2;
+
+  auto hic = std::make_unique<HostIrContainer>();
+  FusionGuard::setCurFusion(hic.get());
+
+  auto index = IrBuilder::create<Val>(DataType::Index);
+  auto start = IrBuilder::create<Val>(kForLoopStart, DataType::Index);
+  auto stop = IrBuilder::create<Val>(kForLoopStop, DataType::Index);
+  auto step = IrBuilder::create<Val>(kForLoopStep, DataType::Index);
+  auto for_loop = IrBuilder::create<kir::ForLoop>(
+      /*IterDomain=*/makeContigConcreteTensor({0})->axis(0), // unused
+      index,
+      start,
+      stop,
+      step,
+      /*vectorize=*/false,
+      /*vectorize_shift=*/nullptr,
+      /*unroll_required=*/false,
+      /*double_buffer_loop_stage=*/DoubleBufferLoopStage::NotApplicable);
+
+  auto fusion = std::make_unique<Fusion>();
+  FusionGuard fg(fusion.get());
+
+  auto arange = iota(
+      IrBuilder::create<Val>(kForLoopStop),
+      IrBuilder::create<Val>(0),
+      IrBuilder::create<Val>(1),
+      DataType::Int);
+  auto i = IrBuilder::create<Val>(DataType::Index);
+  Slice s = {i, add(i, IrBuilder::create<Val>(1)), IrBuilder::create<Val>(1)};
+  auto n = slice(arange, {s});
+  auto acc_in = makeContigConcreteTensor({1}, DataType::Int);
+  auto acc_out = add(acc_in, n);
+
+  fusion->addInput(i);
+  fusion->addInput(acc_in);
+  fusion->addOutput(acc_out);
+
+  FusionGuard::setCurFusion(hic.get());
+
+  IrCloner ir_cloner(hic.get());
+  std::vector<Val*> post_on_stream_inputs = {index, ir_cloner.clone(acc_in)};
+  std::vector<Val*> post_on_stream_outputs = {ir_cloner.clone(acc_in)};
+  auto host_unit = IrBuilder::create<HostUnit>(std::move(fusion));
+  auto post_on_stream = IrBuilder::create<PostOnStream>(
+      host_unit, post_on_stream_inputs, post_on_stream_outputs);
+
+  for_loop->body().push_back(post_on_stream);
+
+  hic->addInput(post_on_stream->inputs().at(1));
+  hic->addOutput(post_on_stream->outputs().at(0));
+  hic->pushBackTopLevelExprs(for_loop);
+
+  HostIrExecutorParams params;
+  auto [use_fusion_executor_cache] = GetParam();
+  if (!use_fusion_executor_cache) {
+    GTEST_SKIP()
+        << "not supported for now because of concretization pb, getting the error: dynamic_tvs.empty() INTERNAL ASSERT FAILED at /opt/pytorch/Fuser/csrc/device_lower/validation.cpp:187, please report a bug with repro script to NVFuser at https://github.com/NVIDIA/Fuser/issues. Tensor with dynamic transform must be concretized before lowering: T1_l[ ?S2{( ( ( -( fmax(0, ( where(( i7 < 0 ), ( i7 + 10 ), i7) )) ) ) + 10 ) + ( ( fmax(( fmax(0, ( where(( i7 < 0 ), ( i7 + 10 ), i7) )) ), ( fmin(10, ( where(( ( i7 + 1 ) < 0 ), ( ( i7 + 1 ) + 10 ), ( i7 + 1 )) )) )) ) - 10 ) )}rf ], T3_g[ ?S4{( ( ( -( fmax(0, ( where(( i7 < 0 ), ( i7 + 10 ), i7) )) ) ) + 10 ) + ( ( fmax(( fmax(0, ( where(( i7 < 0 ), ( i7 + 10 ), i7) )) ), ( fmin(10, ( where(( ( i7 + 1 ) < 0 ), ( ( i7 + 1 ) + 10 ), ( i7 + 1 )) )) )) ) - 10 ) )} ]";
```

Comment:
> concretization pb

What's `pb`?

Reply:
that stands for "problem" :)
I replace "pb" by "issue"